metadata.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "chocolatey-chocolatey",
-  "version": "1.2.1",
+  "name": "puppetlabs-chocolatey",
+  "version": "0.0.1",
   "author": "chocolatey",
   "summary": "The Chocolatey package provider for Puppet",
```

Comment:
Good point.  On review I  definitely want drop the 'The'.  A quick search and I couldn't find any modules that have 'Official Puppet.....' in them.  Even the apache module doesn't.

@ferventcoder Thoughts?